backend/de.metas.document.archive/de.metas.document.archive.base/src/main/java/de/metas/document/archive/spi/impl/DocOutboundArchiveEventListener.java

```diff
@@ -156,6 +150,7 @@ public void onPrintOut(final I_AD_Archive archive,
 		docOutboundLogLineRecord.setStatus(status.getCode());
 
 		save(docOutboundLogLineRecord);
+		docOutboundService.sendMailAutomaticallyIfActive(docOutboundLogLineRecord);
```

Comment:
as discussed, might not be the best...
we also want to support the case of sending mails without printing